content/tutorials/basic-movement.md

```diff
@@ -130,27 +130,27 @@ while True:
 
     # ^^ code from before ^^
 
-    # power up to 70 (from 10)
-    for pwr in range(10, 80, 10):
+    # power up to 0.7 (from 0.1)
+    for pwr in range(0.1, 0.8, 0.1):
```

Comment:
Unfortunately, it's not this simple, `range` only accepts integers. You'll probably need to keep the range and do a divide in code.

Reply:
Good spot - didn't know that was they case, but [that is indeed what the docs say](https://docs.python.org/3/library/stdtypes.html#typesseq-range).